.evergreen.yml

```diff
@@ -6792,11 +6867,22 @@ functions:
           . preload.sh
           curl -sSfL "$ARTIFACT_URL" > mongosh.deb
           dpkg -x mongosh.deb .
+          }
+    - command: shell.exec
+      params:
+        working_dir: src
+        shell: bash
+        script: |
+          set -e
+          set -x
+          {
+          . preload.shortName
```

Comment:
What's preload.shortName?

Reply:
A typo, presumable auto-completed into this … and I guess a reminder that this function is not currently being used. (I don’t see any harm in keeping it around, it matches the rpmextract variant and at some point we may want to use it again)